### PR TITLE
Remove data dog and manifold-performance

### PR DESCRIPTION
**Pull request by TerrySmithDC**
## Reason for change

Removes all references of Data Dog and the the `manifold-performance` component. I'll also be updating our docs to cover deprecating components, how ever in this case the manifold-performance is not in use so we decided to just remove it.

## Testing

Scenario stories were updated now that `manifold-performance` is removed, I validated the auth and metrics logging stories. Metrics logging is working however the new data dog set up for this type of log is "unreleased", after chatting with @domenicrosati I decided to just comment out the story. We can re add a proper link showing  analytics-ingest once that work is more settled and it is possible to provide a direct link to that specific test log. 

## Checklist

- [x] **CHANGELOG**: The **Unreleased** section of CHANGELOG was updated
- [ ] **Prop changes**: [docs][docs] were updated
- [ ] **Prop changes**: E2E tests were updated, testing from the highest level possible
- [ ] **Platform testing**: If this change should be [tested against a platform][platform-testing], has it been?

[docs]: https://ui.sandbox.manifold.co
[platform-testing]: https://app.gitbook.com/@manifold/s/engineering/playbooks/testing-your-code-against-a-platform

### CODE REVIEW

**Comment by dangodev** [comment]
Let’s remove the DataDog reference from `now.json`

Also I think we can remove the `manifold run` command from the `dev` npm-script (as well as the `.manifold.yml.example` file I think)

@gclapperton Since we don’t need the Manifold CLI any more, are there any changes needed for the Docker container?

**Comment by TerrySmithDC** [comment]
Hah! I was actually going to ask you about removing Manifold cli as a dependency, it's caused some issues for set up and seems kinda un necessary.

Looking at `now.json` / `dev` command

File: package.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
     "build:watch": "npm run update-version && stencil build --watch",
     "bundlesize": "npm run build && npx bundlesize",
     "copy-changelog": "node scripts/copy-changelog",
-    "dev": "npm run build:watch & rm -rf dist && wait-on dist/loader/index.cjs.js && manifold run -p manifold-ui-local -- npm run storybook",
+    "dev": "npm run build:watch & rm -rf dist && wait-on dist/loader/index.cjs.js && npm run storybook",
```
Comment (dangodev) [inline package.json]: Nice! One last request: can you remove `.manifold.yml.example`?

File: CHANGELOG.md [hunk 2]
```diff
@@ -11,6 +11,11 @@ adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 - Added `metrics` property to `<manifold-connection>` to opt-in for metric collection (#724)
 
+### Removed
+
+- Data Dog logging removed (#9584)
+- Removed `manifold-performance` component (#9584)
```
Comment (dangodev) [inline CHANGELOG.md]: We’ve been using PR numbers rather than issue numbers in the changelog

File: stories/scenarios.stories.js [hunk 3]
```diff
@@ -27,18 +26,23 @@ storiesOf('Scenarios', module)
 
     return `
       <manifold-connection env="${env}">
```
Comment (dangodev) [inline stories/scenarios.stories.js]: Oh this isn’t part of a PR, but something I noticed in mine: can you add `<manifold-connection metrics>` here? The `metrics` attribute basically replaces the `<manifold-performance>` component.

File: CHANGELOG.md (hunk 2 above)
Comment (TerrySmithDC) [reply CHANGELOG.md]: Oh Sorry! I had the feeling it was too high heh.

**Comment by TerrySmithDC** [comment]
I checked the Dockerfile @dangodev, it looks fine.

**Comment by dangodev** [comment]
@TerrySmithDC not a huge deal, but on next merge please Squash & merge. Just keeps the commit history a bit cleaner.

**Pull request merged**